include/overlay/elements/list_item.hpp

```diff
@@ -40,6 +40,8 @@ namespace tsl::element {
 
         void setClickListener(std::function<bool(s64 keysDown)> clickListener) { this->m_clickListener = clickListener; }
 
+        void updateText(std::string text);

```

Comment:
I'd call this function setText instead probably

Reply:
Renamed, thanks for looking at these PRs!

Reply:
Thanks for making them :)